package.json

```diff
@@ -41,6 +41,7 @@
     "graphql": "^14.0.0 ",
     "graphql-tag": "^2.10.3",
     "nyc": "^14.1.1",
+    "sinon": "^12.0.1",
```

Comment:
I left out the version bump of the `package.json`. It seems to be done in a separate commit after the PR is merged. Let me know if I got this wrong and I'll amend the PR to bump the version.

Reply:
yup that was the right call